stable/selenium/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: selenium
-version: 1.0.3
+version: 1.0.4
 appVersion: 3.14.0
```

Comment:
Since you are updating the chart, @ekirmayer why don't you upgrade to the latest version? 3.141.59?

Reply:
https://github.com/helm/charts/blob/master/stable/selenium/values.yaml#L21

Reply:
@diemol - I thought about it, but don’t you think we should split updating the version from adding features?

Reply:
You are right @ekirmayer, better to do two separate PRs. 👍

Reply:
@diemol - Can you approve the this for merge?